PYTHON_UPGRADE_GUIDE.MD

```diff
@@ -0,0 +1,38 @@
+# Python Upgrade Runbook
+
+So you wanna upgrade PyTorch to support a new Python? Follow these steps in order!
+
+### 0.Before starting upgrade process. Check the package availability
+
+Package availability to validate before starting upgrade process :
+
+1) Manywheel build: Python version is avalilable in either of these sources:
+- Github: https://github.com/python/cpython
+- Python ftp[here](https://www.python.org/ftp/python/). Please consult the [schedule](https://peps.python.org/pep-0719/)
```

Comment:
nit this pep is the 3.13 schedule only. You can use https://devguide.python.org/versions/ as a long-lived link here.